modules/local/cram_filter_align_bwamem2_fixmate_sort.nf

```diff
@@ -18,21 +18,26 @@ process CRAM_FILTER_ALIGN_BWAMEM2_FIXMATE_SORT {
 
     script:
     def args = task.ext.args ?: ''
+    def args1 = task.ext.args1 ?: ''
+    def args2 = task.ext.args2 ?: ''
+    def args3 = task.ext.args3 ?: ''
+    def args4 = task.ext.args4 ?: ''
     def prefix = task.ext.prefix ?: "${meta.id}"
+    // Please be aware one of the tools here required mem = 28 * reference size!!!
     """
     cram_filter -n ${from}-${to} ${cramfile} - | \\
-        samtools fastq -F0xB00 -nt - | \\
+        samtools fastq ${args1} | \\
         bwa-mem2 mem -p ${bwaprefix} -t${task.cpus} -5SPCp -H'${rglines}' - | \\
-        samtools fixmate -mpu - - | \\
-        samtools sort --write-index -l1 -@${task.cpus} -T ${base}_${chunkid}_sort_tmp -o ${prefix}_${base}_${chunkid}_mem.bam -
+        samtools fixmate ${args3} - - | \\
+        samtools sort ${args4} -@${task.cpus} -T ${base}_${chunkid}_sort_tmp -o ${prefix}_${base}_${chunkid}_mem.bam -
 
     cat <<-END_VERSIONS > versions.yml
     "${task.process}":
         samtools: \$(echo \$(samtools --version 2>&1) | sed 's/^.*samtools //; s/Using.*\$//' )
         bwa-mem2: \$(bwa-mem2 --version | sed 's/bwa-mem2 //g')
```

Comment:
```suggestion
        bwa-mem2: \$(bwa-mem2 version | tail -n 1)
        staden_io_lib: \$(ls /usr/local/conda-meta/staden_io_lib-* | cut -d- -f3)
```

Reply:
Done, and added to minimap2 supermodule too